docs/publication_email_requests.md

```diff
@@ -0,0 +1,74 @@
+# Publication Email Request Service (PERS)
+
+The following are sequence diagrams describing the workflow for sending publication email requests with this service. At high-level the steps involved are:
+
+1. Import citations for authors
```

Comment:
Optional rephrasing to improve clarity (but please correct if I'm getting it wrong!):

```
1. Import Publication citations for Authors to database
2. Create emails requesting the Publications based on the citations
3. Review and (optionally) update emails
4. Send emails to Liaisons who will send them to Authors
```